src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -198,7 +200,7 @@ private static (TParameterSyntax?, IParameterSymbol?) GetParameterAtOrdinal(int
                 var operation = semanticModel.GetOperation(coalesceNode, cancellationToken);
                 if (operation is ICoalesceOperation coalesceExpression)
                 {
-                    if (IsParameterReference(coalesceExpression.Value, parameter) &&
+                    if (InitializeParameterHelpersCore.IsParameterReference(coalesceExpression.Value, parameter) &&
```

Comment:
❔ Is this change still necessary with the `using static` above?

Reply:
nope.  cleaned.